dashboard/db/migrate/20210816222729_add_code_review_enabled_to_sections.rb

```diff
@@ -0,0 +1,11 @@
+class AddCodeReviewEnabledToSections < ActiveRecord::Migration[5.2]
+  def up
+    add_column :sections, :code_review_enabled, :boolean
```

Comment:
We seem to use both "code review" and "peer feedback" for this feature so let me know if there's a preference!

Reply:
"Code Review" sounds good to me